app/app.rb

```diff
@@ -0,0 +1,86 @@
+require 'sinatra/base'
+require 'json'
+
+class FakeIdvaas < Sinatra::Application
+  get '/' do
+    'Hello, USA!'
+  end
+
+  get '/idvaas/user/:user_uuid/question/?' do
+    help_image_url = 'http://localhost:8080/idvaas/help-image/' +
+      '22619919-2adb-5825-1cf4-06e1d4a9e11b.jpg'
+    {
+      question: {
+        questionId: '6c354be7-6d0f-46ab-8407-96624a362f17',
+        text: 'What was the priority date for your U.S. immigrant visa?',
+        answers: [
+          {
+            key: 1,
+            text: '06/01/2015'
+          },
+          {
+            key: 2,
+            text: '10/24/2014'
+          },
+          {
+            key: 3,
+            text: '08/15/2014 <-- FAILS TEST'
+          },
+          {
+            key: 4,
+            text: '01/26/2015'
+          },
+          {
+            key: 5,
+            text: 'NONE OF THE ABOVE <-- CORRECT ANSWER'
+          }
+        ],
+        helpMessage: 'This is the help message',
+        helpImageUrl: help_image_url }
+    }.to_json
+  end
+
+  get '/idvaas/user/:user_uuid/?' do
```

Comment:
we can also remove the `/idvaas/` from all of these since it will be hosted on its own domain: `projectName-idv.tld`